### DISCUSSION TITLE
Repositories with long READMEs

### DISCUSSION THREAD

**Discussion by sawa-ko** [community/community#41735] [Mobile]
When a repository has a rather long/big README the application freezes for a few seconds and then crashes.

**Comment by aconsuegra** [comment]
Hi @kaname-png, thanks for reaching out and for your feedback. Can you share a link to a repository that causes the app to freeze and crash? In addition, can you share details about the OS and device that you are using?

**Reply by david-lev** [reply]
Happens to me in yt-dlp
https://github.com/yt-dlp/yt-dlp
The app doesn't crash but the screen gets stuck for a few seconds and the whole device gets very slow.

Samsung Galaxy S21, Android 13
App version 1.126.0

**Comment by jfullstackdev** [comment]
I even opened books length in the past and there was no loading

maybe it's the memory of your computer, check that, if it's too low when you are opening it

maybe your Internet connection or a GitHub extension or your firewall / antivirus

**Reply by sawa-ko** [reply]
This is the category for Github mobile.

**Comment by igorskyflyer** [comment]
Maybe it's not the length of a README but what's in it, like CPU/memory consuming GIFs (even videos) or too many images, e.g. I have a GIF in mine and sometimes the page does stutter.

**Reply by sawa-ko** [reply]
Yes, I can confirm that it is due to the excess of images and gifs that some READMEs have.

Currently I don't have a sample repository, because I forgot to copy its link before reporting.

CC: @aconsuegra

**Reply by sawa-ko** [reply]
However, when a README has a lot of text, for a few seconds the readme is displayed as a black space and then a few seconds later the content appears.
